chapters/ch06.md

```diff
@@ -1,48 +1,48 @@
 
 > Once you define the dev and test sets, your team will be focused on improving dev set performance. Thus, the dev set should reflect the task you want to improve on the most: To do well on all four geographies, and not only two.
 
-Một khi bạn đã phân loại tập phát triển và tập kiểm tra, nhóm của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Bởi vậy, tập phát triển nên đại diện nhiệm vụ bạn muốn cải thiện nhiều nhất: hoạt động tốt trên dữ liệu không chỉ hai mà cả bốn thị trường.
+Một khi bạn đã xác định được tập phát triển và tập kiểm tra, đội nhóm của bạn sẽ tập trung cải thiện kết quả trên tập phát triển. Do đó tập phát triển nên phản ánh được việc mà bạn muốn cải thiện nhiều nhất: hoạt động tốt trên dữ liệu từ không chỉ hai mà cả bốn khu vực địa lý.
 
 > There is a second problem with having different dev and test set distributions: There is a chance that your team will build something that works well on the dev set, only to find that it does poorly on the test set. I’ve seen this result in much frustration and wasted effort. Avoid letting this happen to you.
 
-Vấn đề thứ hai với việc tập phát triển và tập kiểm tra có phân phối khác nhau: Có khả năng nhóm của bạn xây dựng được một mô hình hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy việc này (việc có phân phối khác nhau của hai tập dữ liệu) dẫn đến những kết quả đáng thất vọng và sự lãng phí về công sức. Hãy tránh để điều này xảy ra.
+Vấn đề thứ hai đối với sự phân phối của tập phát triển và tập kiểm tra: Có khả năng đội nhóm của bạn xây dựng được một thứ gì đó hoạt động tốt trên tập phát triển nhưng lại kém trên tập kiểm tra. Tôi đã từng thấy điều này  dẫn đến những kết quả đáng thất vọng và lãng phí công sức ra sao . Hãy tránh để nó xảy ra với bạn.
 
 > As an example, suppose your team develops a system that works well on the dev set but not the test set. If your dev and test sets had come from the same distribution, then you would have a very clear diagnosis of what went wrong: You have overfit the dev set. The obvious cure is to get more dev set data.
 
-Ví dụ, nhóm của bạn phát triển một hệ thống hoạt động tốt trên tập phát triển nhưng kém trên tập kiểm tra. Nếu tập phát triển và tập kiểm tra có cùng một phân phối thì bạn có thể xác định ngay vấn đề: Mô hình của bạn đã overfit tập phát triển. Cách xử lý hiển nhiên nhất đó là bổ sung thêm dữ liệu cho tập phát triển.
+Ví dụ như, đội nhóm của bạn xây dựng một hệ thống hoạt động tốt trên tập phát triển nhưng kém trên tập kiểm tra. Nếu tập phát triển và tập kiểm tra có cùng một tập phân phối thì bạn có thể chuẩn đoán chính xác chỗ nào sai: Tập phát triển của bạn đã bị thiếu. Hiển nhiên là bạn cần bổ sung thêm dữ liệu cho tập phát triển.
```

Comment:
* "đội nhóm"
* "phân phối"
* "overfit" là một thuật ngữ quan trọng trong machine learning, nếu bạn không hiểu rõ nghĩa thì không nên bỏ đi.

Reply:
"team" hay "group" mình nghĩ nên dịch chung là "đội nhóm" để không bị gọn lỏn. "Đội" mang nghĩa là đồng đội, đội ngũ. Cái này không quan trọng lắm, tùy ý bạn.
Mình thấy bạn hay đề cập "làm việc nhóm", nhưng cuốn sách này không phải dạy về cách làm việc theo nhóm mà chỉ đơn thuần đề cập your group (nhóm của bạn - người đang đọc cuốn sách), your team (đội ngũ của bạn)

Reply:
> "team" hay "group" mình nghĩ nên dịch chung là "đội nhóm" để không bị gọn lỏn.

Bạn có thể dịch là "đội" hoặc "nhóm" nhưng phải thống nhất và tôn trọng cách dịch các chương từ đầu. Nếu sửa thì phải sửa toàn bộ. Còn từ ghép "đội nhóm" mình thực sự chưa thấy nơi nào dùng.